pjnat64/src/pjnat64/pj-nat64.c

```diff
@@ -937,12 +940,16 @@ static void patch_contact_ipv6_with_ipv4(pjsip_tx_data *tdata, pjsip_msg *msg)
         }
         //
         if (acc.in4_addr.slen > 0) {
-            PJ_LOG(4, (THIS_FILE, "TX patch Contact addr use account '%.*s':%d (update port only on value >0)",
-                (int)acc.in4_addr.slen, acc.in4_addr.ptr, acc.in4_port
+            PJ_LOG(4, (THIS_FILE, "TX patch Contact addr %.*s -> %.*s (use replace method)",
+                (int)sip_uri->host.slen, sip_uri->host.ptr,
+                (int)acc.in4_addr.slen, acc.in4_addr.ptr
             ));
             sip_uri->host.slen = acc.in4_addr.slen;
             sip_uri->host.ptr = acc.in4_addr.ptr;
-            if (acc.in4_port > 0) {
+            if (acc.in4_port != INVALID_PORT) {
```

Comment:
nit: The check isn't equivalent to previous one, now zero is valid value. Is it ok?

Reply:
This is OK, because `acc.in4_port` initializes with `INVALID_PORT`, and then updates only with values more than 0